Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:alpine
+
+WORKDIR /app
+
+COPY package.json ./
```

Comment:
If you are using npm, we also need the lock file in the repo and added here before the install.